torchdet3d/evaluation/metrics.py

```diff
@@ -1,29 +1,43 @@
 import torch
 
-def compute_average_distance(pred_box, gt_box, num_keypoint=9):
+def compute_average_distance(pred_kp, gt_kp, num_keypoint=9, **kwargs):
     """Computes Average Distance (ADD) metric."""
-    detached_pred_box = pred_box.detach()
```

Comment:
I propose keeping pytorch style: use torch.no_grad() when a gradient is not supposed to be calculated and don't call `detatch()` implicitly.

Reply:
agree